src/main/java/com/google/step/youtube/DescriptionParser.java

```diff
@@ -0,0 +1,14 @@
+package com.google.step.youtube;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.regex.Matcher;
+
+/**
+ * Class for parsing the descriptions of YouTube videos for promotional codes and affiliate links. 
+ */
+public class DescriptionParser {
+
```

Comment:
super nit: say implementation is coming.

Reply:
Added.